### PR TITLE
fix: usage of path, replaced with filepath

### PR DESCRIPTION
**Pull request by SamMHD**
*Issue #23*

*Description of changes:*

- used ```filepath``` package instead of ```path``` package for joining file paths.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: clean.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ func clean(c *ec2macosinit.InitConfig) {
 		}
 		for _, d := range dir {
 			// Remove everything
-			err := os.RemoveAll(path.Join([]string{historyPath, d.Name()}...))
+			err := os.RemoveAll(filepath.Join([]string{historyPath, d.Name()}...))
```
Comment (jahkeup) [inline clean.go:37]: Hrm, not sure why this was dotted, can you fix this usage to avoid the slice literal?

**Review by jahkeup — APPROVED** [review]
LGTM, would like to see one more change to tidy up in the touched code if you have a chance to. Thanks again!

**Review by dpakpak — APPROVED** [review]

**Comment by dpakpak** [comment]
Thanks for fixing this!

File: clean.go (hunk 1 above)
Comment (SamMHD) [reply clean.go]: fixed ✅

**Review by jahkeup — APPROVED** [review]

**Pull request merged**